### ISSUE TITLE
Color states

### ISSUE THREAD

**Issue by bryanjonker-illinois** [web-illinois/ilw-card#5]
![Clickable Card](https://github.com/user-attachments/assets/15db93b3-8b80-4bf3-87e0-87ddf8213555)

Please change the states to match Courtney's document. 

For focus, you can add the following CSS which will reference the variables set in the ilw-global.css document (we are using "outline" instead of any explicit borders)

```
{
    background-color: var(--ilw-link--focus-background-color);
    color: var(--ilw-link--focus-color);
    outline: var(--ilw-link--focus-outline);
}
```

Variables are:
```
--ilw-link--focus-background-color:var(--il-industrial-lighter-4);
--ilw-link--focus-color:var(--il-blue);
--ilw-link--focus-outline:solid var(--il-blue) 1px;
```

**Comment by marvel-uiuc** [comment]
There is one potential issue, best practices for accessible clickable cards say that only one element in the card should be an actual link. Otherwise a screen reader would read the entire card every time the link is referenced.

That means the card never actually receives focus. We could make it appear to receive focus anyway, but that might not be accessible.

The hover is different because it's just a visual, and doesn't affect assistive technology. We should discuss the best approach here.

**Comment by marvel-uiuc** [comment]
We are going to ignore the full card focus for the time being, and just display focus on the link and button elements in the card. That's in line with all accessibility recommendations that I could find dealing with clickable cards.

**Comment by marvel-uiuc** [comment]
Resolved in 84382e51ab73ad7c9ce1db40088df770c7cf9d51.
